client/pom.xml

```diff
@@ -39,8 +39,11 @@
       </plugin>
 
       <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
+       
+		<groupId>org.jacoco</groupId>
+		<artifactId>jacoco-maven-plugin</artifactId>
+		<version>0.7.5.201505241946</version>
```

Comment:
somehow eclipse had issues about the way it was before with this pulgin so its here just for my piece of mind